### ISSUE TITLE
Delta brush pattern on EEG

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10421]
**Preferred term label:**
Delta brush pattern on EEG

**Synonyms**
Periodic high amplitude slow waves with superimposed fast activity

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

- Anti-DPPX antibody-associated encephalitis
- Anti NMDA receptor encephalitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@DLewisSmith 
Could you help us with this term?
I am struggling to understand what part of the EEG hierarchy to add it to

[EEG abnormality HP:0002353](https://hpo.jax.org/browse/term/HP:0002353)

**Comment by DLewisSmith** [comment]
This one is complex as it contains slow and fast components and its ‘normality’ and ictal vs intercostal depends on clinical context (it is normal in some premature neonates). For now, I would suggest being conservative with child of HP:0002353 and stating in the definition that it is a normal finding in some premature neonates.


________________________________

**Comment by MickeySegal** [comment]
I presume "ictal vs intercostal" was meant to be "ictal vs interictal".

As a tangent, I was the first to record such activity in a reductionistic system with only one excitatory neuron: https://pubmed.ncbi.nlm.nih.gov/1646871/

**Comment by DLewisSmith** [comment]
Yes. Sorry, trying to catch up on emails on the
run by phone. We are used to muscle artifact on EEG but I’m yet to see intercostal signal 😉

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001051
- label: Delta brush pattern on EEG
- parents: [EEG abnormality (HP:0002353)](https://hpo.jax.org/app/browse/term/HP:0002353)
- pmid: PMID:30214965
- def: Delta brushes are transient patterns characterised by a slow delta wave with superimposed fast activity.
- comment: Delta brushes are the hallmark of the EEG of premature infants.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10421
- synonyms:
